nautobot/docs/development/index.md

```diff
@@ -223,3 +223,20 @@ Feature requests will follow our published workflow from inbound triage to ultim
     * All unit tests pass successfully
     * PEP 8 compliance is enforced, with the exception that lines may be
       greater than 80 characters in length
+    * At least one [changelog fragment](#creating-changelog-fragments) has
+      been included in the feature branch
+
+#### Creating Changelog Fragments
+
+All pull requests to `next` or `develop` must include a changelog fragment file in the `./changes` directory. To create a fragment, use your github issue number and fragment type as the filename. For example, `2362.added`. Valid fragment types are `added`, `changed`, `deprecated`, `fixed`, `removed`, and `security`. The change summary is added to the file in plain text. Change summaries should be complete sentences, starting with a capital letter and ending with a period, and be in past tense.
```

Comment:
Do we need a note about whitespace in the change summary? Are accidental leading/trailing whitespace automatically stripped, or will they make `markdownlint` unhappy when the changelog gets rendered? Is a multi-line change summary permissible, or should it always be a single line?

Reply:
Leading/trailing whitespace is stripped. Multiple lines don't work with the current template so I added a note about only using a single line.